code/modules/reagents/reagent_dispenser.dm

```diff
@@ -68,6 +75,22 @@
 	else
 		qdel(src)
 
+/obj/structure/reagent_dispensers/wrench_act(mob/living/user, obj/item/tool)
+	. = ..()
+	if(!openable)
+		return FALSE
+	leaking = !leaking
+	to_chat(user, span_notice("You [leaking ? "open" : "close"] a small tap on the back of [src]."))
```

Comment:
balloon alert this!
balloon alerts are:
* shorter
* without periods unless an exclamation point for failure or ... for an in progress action
* do not capitalize